src/main/java/uk/gov/companieshouse/presenter/account/service/api/ApiClientServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.companieshouse.presenter.account.service.api;
+
+import consumer.exception.NonRetryableErrorException;
+import consumer.exception.RetryableErrorException;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.presenteraccount.request.GetPresenterAccountDetails;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.presenteraccount.PresenterAccountDetailsApi;
+import uk.gov.companieshouse.logging.Logger;
+
+@Primary
+@Service
+public class ApiClientServiceImpl implements ApiClientService {
+    private final InternalApiClient  internalApiClient;
+    private final Logger logger;
+
+    public ApiClientServiceImpl(InternalApiClient internalApiClient, Logger logger) {
+        this.internalApiClient = internalApiClient;
+        this.logger = logger;
+    }
+
+    @Override
+    public ApiResponse<PresenterAccountDetailsApi> getPresenterAccountDetails(String presenterId){
```

Comment:
I would return the details directly here
```suggestion
    public PresenterAccountDetailsApi getPresenterAccountDetails(String presenterId){
```

Reply:
suggestion accepted.